packages/runtime/src/interpreter/interpreter.ts

```diff
@@ -55,7 +55,7 @@ export class Interpreter {
 	length: number; // total length of 'compiled' TEAL code
 	// local stores for a transaction.
 	bytecblock: Uint8Array[];
-	intcblock: BigInt[];
+	intcblock: bigint[];
```

Comment:
I don't think we have bigint in small case. Please check this.

Reply:
bigint small case is the type and BigInt is the constructor of the type

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt

Reply:
ok